CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ endif()
 
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Provide -stdlib=libc++ if necessary for cstdint to be defined
```

Comment:
Oh, I get it. The OS X builders are soo old that libc++ isn't the default. I agree with @samantharitter's suggestion to do this externally via CMAKE_CXX_FLAGS in the YAML file.